_episodes/first_example_recipe.md

```diff
@@ -0,0 +1,332 @@
+    - model grid (for CMIP6 data only, key: grid)
+    - alias (key: alias; use the alias for e.g. a more human readable name)
+
+
+  - preprocessors: lines 25-28
+
+    The definition for different preprocessors or combinations.
+    If no preprocessing is needed, the preprocessor can be set to an empty python dictionary (`{}`).
+    Here, we produce annual means. The preprocessor is called with its name (here: prep_timeseries), later in the diagnostic (line 39).
+    (See episode #5 LINK for more details.)
+
+
+  - diagnostic section: lines 30-42
+
+    The information of which diagnostic script to run with which variables.
+    The diagnostics section has some indents that are free to call.
+    - the first indent (here: diag_timeseries_temperature) is the diagnostic’s name (a string without whitespace), used for setting up the respective directories
+    - description: a short description of the diagnostic
+    - variables: a definition of all variables that are used in this diagnostic
+    - the next indent (here: timeseries_variable) is the variables’ names (a string without whitespace) for the diagnostic to use
+    - short_name: the variable name as listed in the dataset
+    - preprocessor: the preprocessor(s) applied to the variable before running the diagnostic
+    - scripts: a definition of all scripts that are used in this diagnostic
+    - the next indent (here: timeseries_diag) is the scripts’ names (a string without whitespace) for the script to use
+    - script: a executable script with a directory relative to the `esmvaltool/diag_scripts/` directory
+
+> ## Please answer the following questions:
+> What is the short_name of the variable being analyzed?
+>
+> What is the diagnostic script being used?
+>
+> How many years of data are being analyzed?
+>
+> What do you think running this recipe will produce?
+{: .challenge}
+
+> ## What is the short_name of the variable being analyzed?
+> thetaoga -  Global Average Sea Water Potential Temperature
+{: .solution}
+
+> ## What is the diagnostic script being used?
+> `ocean/diagnostic_timeseries.py`
+{: .solution}
+
+> ## How many years of data are being analyzed?
+> 1859 to 2005, that is 147 years.
+{: .solution}
+
+> ## What do you think running this recipe will produce?
+> A time series plot of thetaoga with increements of 1 year.
+{: .solution}
+
+> ## Not all parts of the recipe are mandatory
+> Some functionalities of the example recipe are mandatory, while others are not. E.g., if you miss any of the documentation information, the call will break.
+{: .callout}
+
+> ## Running ESMValTool
+>
+> Use the command:
+> ~~~bash
+> esmvaltool -c ./path_to_file/user-config.yml ./path_to_file/recipe_example.yml
+> ~~~
+>
+> Follow the terminal guiding you through the subprocesses that are running. Can you find where the preprocessor and the diagnostic are starting? Which one took longer to process?
+{: .challenge}
+
+> ## Exemplary output
+> FIXME
```

Comment:
The FIXMEs have to be filled out by someone who can run the recipe on JASMIN (I do not have an account).

It is solutions to the challenges and they should be as close to the expected results as possible.

Reply:
Based on discussion @workshop, I include it based on my output.